Scoop/Supporting/Keys.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 
 struct Keys {
     static let googleClientID = Keys.keyDict["CLIENT_ID"] as? String ?? ""
+    static let scoopedServer = Keys.keyDict["SCOOPED_SERVER_URL"] as? String ?? ""
 
 //    private static let googleServiceDict: NSDictionary = {
```

Comment:
Do we need this commented out code? If so can we document why? If not can we delete it?

Reply:
I think it was there since you need to use it to access the googleClientID, but since that is already placed in keysDict it is not needed